bin/pg_repack.c

```diff
@@ -487,6 +529,12 @@ preliminary_checks(char *errbuf, size_t errsize){
 		goto cleanup;
 	}
 
+	if (check_systemtables()) {
```

Comment:
Maybe it would make sense to make this check inside `repack_one_table` and `repack_table_indexes` only? That way this check won't interrupt pg_repack completely, and it will skip only system tables.